src/jats/importer/jats-body-dom-parser.ts

```diff
@@ -702,6 +702,33 @@ const nodes: NodeRule[] = [
     node: 'caption_title',
     context: 'figcaption/',
   },
+  {
```

Comment:
Shouldn't we remove the old way we did that around src/jats/importer/parse-jats-article.ts:93?